app/version.js

```diff
@@ -0,0 +1,18 @@
+const fs = require('fs');
+
+function getVersionData() {
+  return new Promise((resolve, reject) => {
+    fs.readFile('version.json', 'utf8', function(err, data) {
+      if (err) {
+        reject(err);
+      }
+      try {
+        resolve(JSON.parse(data));
+      } catch(e) {
+        reject(e);
+      }
+    });
+  });
+}
+
+module.exports = getVersionData;
```

Comment:
Although not promise-based, why wrap in a Promise and `fs.readFile()` versus just:

```js
module.exports = require('./version.json');
```

(Or require version.json directly from app/service.js)